### PR TITLE
Fix the encoding issue of DFM extension

### PR DESCRIPTION
**Pull request by qinezh**
#933

### CODE REVIEW

**Comment by qinezh** [comment]
@928PJY

File: src/VscPreviewExtension/package.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
   "displayName": "DfmPreview",
   "description": "Preview the DocFX Flavored Markdown and token tree in Visual Studio Code",
   "version": "0.0.21",
-  "publisher": "docfxsvc",
```
Comment (hellosnow) [inline src/VscPreviewExtension/package.json]: >docfxsvc [](start = 16, length = 8)

docfxsvc is our vso user...

Comment (qinezh) [inline src/VscPreviewExtension/package.json]: 囧rz...

---
In reply to: [91677045](https://github.com/dotnet/docfx/pull/985#discussion_r91677045) [](ancestors = 91677045)

File: src/VscPreviewExtension/src/childProcessHost.ts [hunk 2]
```diff
@@ -19,9 +22,9 @@ export class ChildProcessHost {
     constructor(context: ExtensionContext) {
         // TODO: make path configurable
         let exePath = context.asAbsolutePath("./DfmParse/Microsoft.DocAsCode.Dfm.VscPreview.exe");
-        this._spawn = childProcess.spawn(exePath);
+        this._spawn = Common.spawn('chcp 65001 > NUL & "' + exePath + '"', {});
```
Comment (vicancy) [inline src/VscPreviewExtension/src/childProcessHost.ts]: add some comments

**Pull request merged**